examples/fastyboy.rs

```diff
@@ -37,20 +43,19 @@ pub fn get_pid(process_name: &str) -> process_memory::Pid {
         th32ParentProcessID: 0,
         pcPriClassBase: 0,
         dwFlags: 0,
-        szExeFile: [0; winapi::shared::minwindef::MAX_PATH],
+        szExeFile: [windows::CHAR(0); windows::MAX_PATH as usize],
     };
-    let snapshot: winapi::um::winnt::HANDLE;
     unsafe {
-        snapshot = winapi::um::tlhelp32::CreateToolhelp32Snapshot(
-            winapi::um::tlhelp32::TH32CS_SNAPPROCESS,
-            0,
-        );
-        if winapi::um::tlhelp32::Process32First(snapshot, &mut entry)
-            == winapi::shared::minwindef::TRUE
+        // On Error return 0 as the pid. Maybe this function should instead return itself a Result to indicate if a pid has been found?
```

Comment:
I think it may be worth leaving this comment in, but you may need to run `rustfmt` on this file - the comment is too long for a single line.

Reply:
I actually did run rustfmt on the whole project before making the PR and just tried it again. For some reason rustfmt does not change this comment nor anything else in the file. Verbose output shows that cargo fmt did actually run on fastyboy.rs. 

I wrapped the comment manually for now.